.github/workflows/release-brew.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       run: brew install dafny || echo DONE
     - name: Get Z3
       run: |
-        wget -q https://github.com/Z3Prover/z3/releases/download/z3-4.8.4/z3-4.8.4.d6df51951f4c-x64-osx-10.14.1.zip
+        wget -q https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-osx-10.14.2.zip
```

Comment:
brew install dafny gets Z3 for itself. We should not need to do this here

Reply:
Should I remove all mention of Z3 in this file?

This file also explicitly mentions the prover path to z3 in the "Check" and "Check bad" steps. Can they be removed as well (since I expect Dafny/Boogie to find Z3 in the relative location indicated)?